src/index.js

```diff
@@ -1,46 +1,91 @@
+import _ from 'lodash';
 import './style.css';
-import changeState from './markStatus';
-
-const listContainer = document.querySelector('.list');
-
-const tasks = [
-  {
-    description: 'to do housework',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'go to gym',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'finish Microverse Project',
-    completed: false,
-    index: 3,
-  },
-];
-
-const saveTaskToLocal = (value) => {
+// eslint-disable-next-line
+import { changeState } from './events.js';
+
+export const listContainer = document.querySelector('.list');
+// eslint-disable-next-line
+let tasks = [];
+export { tasks };
+
+export const saveTaskToLocal = (value) => {
   localStorage.setItem('task', JSON.stringify(value));
 };
 
-saveTaskToLocal(tasks);
+const getTaskFromLocal = () => {
+  const dataFromLocal = JSON.parse(localStorage.getItem('task'));
+  if (dataFromLocal) {
+    _.forEach(dataFromLocal, (data, i) => {
+      data.index = i + 1;
+      tasks.push(data);
+    });
+  }
+};
 
-const getTaskFromLocal = () => JSON.parse(localStorage.getItem('task'));
+getTaskFromLocal();
 
-const populateList = (values) => {
-  values.forEach((toDo, i) => {
+const updateTasks = (data) => {
+  tasks = data;
+};
+
+export const removeAll = () => {
```

Comment:
- [ ] This function name doesn't reflect exactly its purpose (Remove all **completed** tasks) and rather bring some confusion. Let's give a meaningful name to this function to keep your code clean and simple.

Reply:
Done